clang/lib/APINotes/APINotesWriter.cpp

```diff
@@ -1266,11 +1266,11 @@ class CommonTypeTableInfo
 class TagTableInfo : public CommonTypeTableInfo<TagTableInfo, TagInfo> {
 public:
   unsigned getUnversionedInfoSize(const TagInfo &TI) {
-    return 2 + (TI.SwiftImportAs ? TI.SwiftImportAs->size() : 0) +
-           2 + (TI.SwiftRetainOp ? TI.SwiftRetainOp->size() : 0) +
-           2 + (TI.SwiftReleaseOp ? TI.SwiftReleaseOp->size() : 0) +
-           2 + (TI.SwiftConformance ? TI.SwiftConformance->size() : 0) +
-           2 + getCommonTypeInfoSize(TI);
+    return 2 + (TI.SwiftImportAs ? TI.SwiftImportAs->size() : 0) + 2 +
+           (TI.SwiftRetainOp ? TI.SwiftRetainOp->size() : 0) + 2 +
+           (TI.SwiftReleaseOp ? TI.SwiftReleaseOp->size() : 0) + 2 +
+           (TI.SwiftConformance ? TI.SwiftConformance->size() : 0) + 3 +
+           getCommonTypeInfoSize(TI);
```

Comment:
Last time I touched this code, I think @compnerd preferred to not clang-format this, but I don't have a super strong opinion on this.

Reply:
Yeah, I think that the grouping helps understand what is being added up (aka, don't clang-format this).

Reply:
I added clang format directives to prevent this from happening in the future.